src/main/java/eu/dissco/backend/security/WebSecurityConfig.java

```diff
@@ -23,6 +23,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
     http.authorizeHttpRequests(authorizeHttpRequests -> authorizeHttpRequests
         .requestMatchers(EndpointRequest.to(HealthEndpoint.class)).permitAll()
         .requestMatchers(HttpMethod.GET, "**").permitAll()
+        .requestMatchers(HttpMethod.POST, "*/v1/annotations/batch").hasRole("dissco-web-batch-annotations")
```

Comment:
I like the annotations above the endpoint better, as it is clearly visible. You can just in the Controller do a `@hasRole("dissco-web-batch-annotations")` and it will do the same thing. I would say for general things such as all Get endpoints are open we put them here but for roles for specific endpoints we put them on top of the method. What do you think?